src/assets/i18n/es.json

```diff
@@ -63,7 +63,7 @@
     "SEND_DATA": "Enviar datos",
     "AUTOCOMPLETE_HINT": "Cuando esté disponible, use las flechas hacia arriba y hacia abajo para revisar e ingresar para seleccionar.",
     "LOADING": "Cargando",
-    "TRACT_SINGULAR": "Sección Censal",
+    "TRACT_SINGULAR": "Sección Censal", 
```

Comment:
It looks like this is just a whitespace addition?

Reply:
whoops, will drop